src/countOccurrences.js

```diff
@@ -18,7 +18,17 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let counter = 0;
+  const phraseModified = phrase.toLocaleLowerCase();
+  const letterModified = letter.toLowerCase();
```

Comment:
используй toLowerCase вместо toLocal
```suggestion
  const phraseLowerCase = phrase.toLocaleLowerCase();
  const letterLowerCase = letter.toLowerCase();
```
обозначай переменные по имени согласно из содержимому
если я встречу переменную в коде, я должен понять что там